Fix bfs building on other architectures such as ARM.

Bug reported and build fix provided by Adam Sampson

-ck

---
 kernel/sched/bfs.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-4.7-ck5/kernel/sched/bfs.c
===================================================================
--- linux-4.7-ck5.orig/kernel/sched/bfs.c	2016-09-23 08:33:00.962719801 +1000
+++ linux-4.7-ck5/kernel/sched/bfs.c	2016-09-23 08:33:00.959719820 +1000
@@ -35,7 +35,7 @@
 #include <linux/init.h>
 #include <asm/uaccess.h>
 #include <linux/highmem.h>
-#include <asm/mmu_context.h>
+#include <linux/mmu_context.h>
 #include <linux/interrupt.h>
 #include <linux/capability.h>
 #include <linux/completion.h>
